pkg/services/signingkeys/signingkeys.go

```diff
@@ -0,0 +1,32 @@
+// Package signingkeys implements the SigningKeys service which is responsible for managing
+// the signing keys used to sign and verify JWT tokens.
+//
+// The service is under active development and is not yet ready for production use.
+//
+// Currently, it only supports RSA keys and the keys are stored in memory.
+
+package signingkeys
+
+import (
+	"crypto"
+
+	"github.com/go-jose/go-jose/v3"
+)
+
+// Service provides functionality for managing signing keys used to sign and verify JWT tokens.
+//
+// The service is under active development and is not yet ready for production use.
+type Service interface {
```

Comment:
Since this is experimental and unstable I understand not doing it, but it's good practice to add a FakeService Implementation for other services to use in tests